### DISCUSSION TITLE
I cannot fully load the @elastic/ui on Tauri app

### DISCUSSION THREAD

**Discussion by hckhanh** [tauri-apps/tauri#7370] [Q&A] [answered]
When I try to load `@elastic/ui`, I get this error and the icons were not fully loaded.

Vite: `v4.2.3`
Tauri: `v1.4`

![image](https://github.com/tauri-apps/tauri/assets/6380436/3f5c224a-1f89-4220-8f2e-f3a64564e4a1)

**Comment by hckhanh** [comment]
Can you check it @FabianLars?

**Reply by FabianLars** [reply, marked as answer]
I can see the same issue in my normal browser so it seems to be a general problem with their package (or the combination with vite) and not something with tauri. Is it possible to reach out to the elastic team instead? As an additional pointer, in firefox i get an error about an invalid mime type `""`.

**Reply by hckhanh** [reply]
thank you @FabianLars
